commerce_coordinator/apps/commercetools/views.py

```diff
@@ -132,20 +132,31 @@ def post(self, request):
         message_details = OrderReturnedViewMessageInputSerializer(data=input_data)
         message_details.is_valid(raise_exception=True)
         order_id = message_details.data['order_id']
-        return_line_item_return_id = message_details.get_return_line_item_return_id()
-        message_id = message_details.data['message_id']
 
-        if self._is_running(tag, order_id):  # pragma no cover
-            self.meta_should_mark_not_running = False
-            return Response(status=status.HTTP_200_OK)
-        else:
-            self.mark_running(tag, order_id)
-
-        fulfill_order_returned_signal.send_robust(
-            sender=self,
-            order_id=order_id,
-            return_line_item_return_id=return_line_item_return_id,
-            message_id=message_id
-        )
+        return_items = message_details.get_return_line_items()
+        for item in return_items:
+            return_line_item_return_id = self.get_return_line_item_return_id(item)
+            return_line_item_id = self.get_return_line_item_id(item)
+            message_id = message_details.data['message_id']
+
+            if self._is_running(tag, f'{return_line_item_return_id}1'):  # pragma no cover
```

Comment:
why `1` at end?

Reply:
i was testing with cache key updates. will remove it